package ru.yandex.practicum.filmorate.storage.mpa;

import ru.yandex.practicum.filmorate.model.Mpa;

import java.util.List;

public interface MpaStorage {

    List<Mpa> findAll();

    Mpa getById(Integer id);

    boolean existsById(Integer id);
}